### ISSUE TITLE
Bunch doesn't follow bunch directory symlink

### ISSUE THREAD

**Issue by paulroub** [ttscoff/bunch#246]
I have a dotfiles repo, managed by [dotbot](https://github.com/anishathalye/dotbot), that includes my bunches.  

dotbot links my `~/dotfiles/bunches` to `~/bunches`. 

```sh
$ ls -l bunches
lrwxr-xr-x  1 paulroub  staff  32 Aug 15 09:04 bunches -> /Users/paulroub/dotfiles/bunches

$ ls  ~/dotfiles/bunches
release.bunch present.bunch # etc
```

If I configure bunch to look into `~/dotfiles/bunches`, all is well. If I instead point it to `~/bunches`, it finds nothing.

**Comment by ttscoff** [comment]
Interesting. I can see how this would be beneficial and will look into 
what might be causing the issue.

**Comment by paulroub** [comment]
The workaround is painfully easy and obvious, of course, so it's far from a high priority concern.

**Comment by ttscoff** [comment]
I think I have this working. When I tested it I got a different result: trying to select a symlink/alias in the open dialog immediately resolved to the source folder. So it worked, it was just impossible to actually use a symlink. If I have it NOT resolve the alias, then the contentsOfDirectoryAtURL returns empty. So I decided to just have it resolve the URL in memory before parsing the folder, which seems to be working. Will test a bit more before putting it out.
